stack.yaml

```diff
@@ -37,7 +37,18 @@ packages:
 # Dependency packages to be pulled from upstream that are not in the resolver
 # using the same syntax as the packages field.
 # (e.g., acme-missiles-0.3)
-# extra-deps: []
+extra-deps:
+- pandoc-2.11.2
+- pandoc-types-1.22
+- hslua-1.1.2
+- jira-wiki-markup-1.3.2
+- commonmark-0.1.1.2
+- commonmark-extensions-0.2.0.4
+- commonmark-pandoc-0.2.0.1
+- citeproc-0.2
+- rfc5051-0.2
+- skylighting-0.10.1
+- skylighting-core-0.10.1
```

Comment:
Ideally we'd specify these bounds in `package.yaml`, as in #4. Let me try to play around with the bounds.

Reply:
I'm not too familiar with haskell dependency management tbh, so sorry if that's not the best way to do it. Back when I wrote this, I think there wasn't any resolver which included all of the dependencies without using "extra-deps". It might be different now though.

Reply:
I think you're right. See my comment below describing a proposed approach.